# Building a native, statically-linked insmod can result in ld failing because
# it can't find a static libc.  A native, static insmod is unnecessary anyway,
# so this patch just removes it.
#
# - jdike@linux.intel.com
#
# Change to a simpler method by just disabling static link flag, since even
# removing those lines don't prevent insmod.static from compile
#
# by Kevin Tian <kevin.tian@intel.com>, 2010-07-21

Upstream-Status: Pending

Index: module-init-tools-3.2.2/Makefile.am
===================================================================
--- module-init-tools-3.2.2.orig/Makefile.am	2010-05-26 15:43:16.000000000 -0400
+++ module-init-tools-3.2.2/Makefile.am	2010-05-26 16:04:06.232990785 -0400
@@ -10,7 +10,6 @@
 modindex_SOURCES = modindex.c zlibsupport.c zlibsupport.h testing.h
 
 insmod_static_SOURCES = insmod.c
-insmod_static_LDFLAGS = -static
 # We don't want the $(zlib_flags) here: that makes a dynamic executable
 insmod_static_LDADD = libmodtools.a
 
